features/user_can_pay_for_order.feature

```diff
@@ -0,0 +1,73 @@
+Feature: User can pay for order
```

Comment:
Since you're using `@javascript` on all scenario you can add it to the top of the feature then it'll apply to all scenario

```gherkin
@javascript
Feature: ...
...
```

Reply:
I can update that, thank you

Reply:
👍